ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -200,6 +200,8 @@ def ocs_install_verification(
         f"{storage_cluster_name}-cephfs",
         f"{storage_cluster_name}-ceph-rbd",
     }
+    if float(config.ENV_DATA["ocs_version"]) >= 4.8:
```

Comment:
We need to use now the `semantic_version` due to this issue https://github.com/red-hat-storage/ocs-ci/issues/4066.
So change the line to `Version.coerce(ocs_version) >= Version.coerce("4.8")`

Reply:
Thanks @yitzhak12 for noticing. Fixed.